components/contexts/ThemeContext.tsx

```diff
@@ -0,0 +1,59 @@
+import { createContext, FC, useMemo } from 'react'
```

Comment:
@nbonatsakis can you look at this? Is this what you had in mind for specifying light vs dark theme? See the next file down too.

Reply:
I think that's right. How are you consuming this? I was thinking we'd have a `useTheme` hook that we could just call in any component. Just shorthand for the raw react context stuff.

Reply:
Yes absolutely right, we are making use of the `useTheme()` hook everywhere. `ThemeContext.tsx` is the file where we are creating the context that exports the `Provider` and `Context` itself. Here are some places where I have used the `useTheme()` hook.

https://github.com/explanation/lava-components/blob/b56dd7bb3567d26822076dad08abafb3a6a45ee5/components/atoms/Text/Text.tsx#L25-L34
https://github.com/explanation/lava-components/blob/b56dd7bb3567d26822076dad08abafb3a6a45ee5/components/atoms/Label/Label.tsx#L12-L39
https://github.com/explanation/lava-components/blob/b56dd7bb3567d26822076dad08abafb3a6a45ee5/components/molecules/GameCard/GameCard.tsx#L52-L75